python/cudf/dataframe/columnops.py

```diff
@@ -88,7 +88,7 @@ def sort_by_values(self, ascending):
     def find_and_replace(self, to_replace, values):
         raise NotImplementedError
 
-    def fillna(self, fill_value):
+    def fillna(self, fill_value, inplace):
```

Comment:
The function call from `series.fillna` requires an `inplace` argument
See:
https://github.com/rapidsai/cudf/blob/afa2114b6153cf6800affe6a80a7a34d856e2aa0/python/cudf/dataframe/series.py#L678